wp-content/themes/core/blocks/tribe/example/block.json

```diff
@@ -1,17 +0,0 @@
-{
-	"$schema": "https://schemas.wp.org/trunk/block.json",
-	"apiVersion": 2,
-	"name": "tribe/example",
-	"version": "0.1.0",
-	"title": "Example Block",
-	"category": "tribe-custom",
-	"icon": "smiley",
-	"description": "Example block scaffolded with Create Block tool.",
-	"supports": {
-		"html": false
-	},
-	"textdomain": "example",
-	"editorScript": "file:./index.js",
-	"editorStyle": "file:./index.css",
-	"style": "file:./style-index.css"
```

Comment:
I'd prefer to leave the example custom block definition in Moose as we will eventually get to writing our own fully-custom blocks. :)

Reply:
In addition to this block, it'd be great to have an example block utilizing ACF. As we kickoff more of these projects, that will ideally be our go-to solution for custom blocks (at the moment). So we should ensure that all devs have a good starting-point for that.

Reply:
I've added the `tribe/example` block and added an ACF example block under `blocks/acf/example` as well. 💪